### PR TITLE
Update to5.16

### PR DESCRIPTION
**Pull request by whazenberg**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ git:
 branches:
   only:
     - master
+    - fix-shadowing
```
Comment (whazenberg) [inline .travis.yml:19]: I've noticed that the branch also still exists, keep or remove?

File: .travis.yml [hunk 2]
```diff
@@ -32,9 +33,9 @@ env:
     - PIP_CACHE=$HOME/.cache/pip
     - SDK_TESTING=true
     - BUNDLE_PATH=$TRAVIS_BUILD_DIR/vendor/cache
-    - DD_AGENT_BRANCH=master
+    - DD_AGENT_BRANCH=updateTo5.16
```
Comment (whazenberg) [inline .travis.yml:36]: Usual

File: Rakefile [hunk 3]
```diff
@@ -12,6 +12,8 @@ unless ENV['CI']
   ENV['NOSE_FILTER'] = ENV['NOSE_FILTER'] || 'not windows'
   ENV['RUN_VENV'] = 'true'
   ENV['SDK_TESTING'] = 'true'
+  ENV['DD_AGENT_REPO'] = 'https://github.com/StackVista/sts-agent.git'
+  ENV['DD_AGENT_BRANCH'] = 'updateTo5.16'
```
Comment (whazenberg) [inline Rakefile:16]: usual

File: kubernetes_topology/conf.yaml.example [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  #      - optional_tag1
+  #      - optional_tag2
+
+instances:
+  # The kubernetes topology check retrieves topology from the kubernetes api.
+  # By default we will assume we're running under docker and will use the address
+  # of the default router to reach the kubernetes api.
+  #
+  # To override, e.g. in the case of a standalone instance, use the following:
+  #
```
Comment (whazenberg) [inline kubernetes_topology/conf.yaml.example:12]: use the following: <nothing>?
Remove or add the missing setting?

File: kubernetes_topology/manifest.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "maintainer": "info@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline kubernetes_topology/manifest.json:5]: for the min and max versions, what is checked here; their version or our version number?

File: mesos_master_topology/manifest.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "help@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline mesos_master_topology/manifest.json:5]: same note about the version being checked

File: splunk_event/manifest.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "help@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline splunk_event/manifest.json:5]: same note about the version being checked

File: splunk_metric/manifest.json [hunk 8]
```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "info@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline splunk_metric/manifest.json:5]: same note about the version being checked

File: splunk_topology/manifest.json [hunk 9]
```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "help@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline splunk_topology/manifest.json:5]: same note about the version being checked

File: ucmdb_file/manifest.json [hunk 10]
```diff
@@ -0,0 +1,15 @@
+{
+  "maintainer": "info@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```
Comment (whazenberg) [inline ucmdb_file/manifest.json:5]: same note about the version being checked

File: .travis.yml (hunk 1 above)
Comment (craffit) [reply .travis.yml:19]: removed

File: kubernetes_topology/manifest.json (hunk 5 above)
Comment (craffit) [reply kubernetes_topology/manifest.json:5]: Our version, this is only relevant when building checks separately from the main agent

File: kubernetes_topology/conf.yaml.example (hunk 4 above)
Comment (craffit) [reply kubernetes_topology/conf.yaml.example:12]: made wording clearer

**Pull request closed without merging**